demisto_sdk/commands/common/constants.py

```diff
@@ -286,6 +286,7 @@
 PACKS_SCRIPT_README_REGEX = fr'{SCRIPT_DIR_REGEX}/README.md$'
 
 PACKS_SCRIPT_NON_SPLIT_BASE_REGEX = fr'{SCRIPTS_DIR_REGEX}/script-([^\\/]+)'
+PACKS_SCRIPT_TEST_PLAYBOOK = fr'{PACK_DIR_REGEX}\/{TEST_PLAYBOOKS_DIR}/script-([^\\/]+).yml$'
```

Comment:
no need to escape slashes.
```suggestion
PACKS_SCRIPT_TEST_PLAYBOOK = fr'{PACK_DIR_REGEX}/{TEST_PLAYBOOKS_DIR}/script-([^\\/]+).yml$'
```